beard/clustering/tests/test_wrappers.py

```diff
@@ -27,91 +27,223 @@
 from ..wrappers import ScipyHierarchicalClustering
 from beard.metrics import b3_f_score
 
-from sklearn.metrics import silhouette_score
+from beard.metrics import silhouette_score
 from functools import partial
 
 
 # Test wrapper of Scipy hierarchical clustering
-
 @pytest.fixture
-def testData():
+def test_data():
     """Preparing a test data."""
     random_state = check_random_state(42)
 
     return make_blobs(centers=4, shuffle=False, random_state=random_state)
 
 
-def test_unsupervised(testData):
-    """Unsupervised clustering."""
-    X, y = testData
+@pytest.fixture
+def affinity_data():
```

Comment:
There is a lot of repeated code here. Cant we factor out the generation of data?